manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
```

Comment:
These imports are minimal and sufficient for the task. Ensure they are necessary for the scope of this script, which they are in this case.